docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+
+services:
+    db:
+        image: "postgres"
+        container_name: "db"
+        environment: 
+            - POSTGRES_PASSWORD=django 
+            - POSTGRES_USER=django
+            - POSTGRES_DB=healthcheck
+        # ports:
+        #     - "5432:5432"
+        volumes:
+            - ./dbdata:/var/lib/postgresql/data
+    django:
+        build:
+            context: .
+            dockerfile: ./Dockerfile
+        command: sh -c "python manage.py collectstatic --noinput && python manage.py runserver 0.0.0.0:8000"
```

Comment:
I think we should rather do the `collectstatic` as a step in the dockerfile, rather than doing it at runtime. Rather use the default entrypoint script in the docker container that handles migrations, nginx, etc.

Reply:
It's already done in the dockerfile. Will be resolved in new commit.